src/resources/assets/js/createContainer.vue

```diff
@@ -186,29 +187,34 @@ export default {
             let newFileNames = newFiles.map(f => f.name);
             files = files.filter(f => f.saved || !newFileNames.includes(f.name));
 
-            this.selectedDirectory.files = files;
             newFiles.forEach((file) => {
                 file._status = reactive({
                     failed: false,
                     info: false,
                 });
                 files.push(file);
+            });
 
-                if (file.type == "image/tiff") {
-                    let allFiles = this.extractFiles(this.rootDirectory);
-                    let fileName = file.name;
-                    let dirname = allFiles.find(entry => entry.file.name === fileName);
-                    let path = "/" + dirname.prefix;
-                    this.getHightWidth(file, path, (rightSize) => {
-                        if (!rightSize) {
+            this.selectedDirectory.files = files;
+
+            let allFiles = this.extractFiles(this.rootDirectory);
+            newFiles.filter(f => f.type === "image/tiff").forEach((file) => {
+                let fileName = file.name;
+                let dirname = allFiles.find(entry => entry.file.name === fileName);
+                let path = "/" + dirname.prefix;
+                this.getHeightWidth(file).then((size) => {
+                    if (size.width < this.threshold || size.height < this.threshold) {
+                        if (file.size < 1000000000) {
+                            this.hasRejectedTiff = true;
+                        }
+                        if (size.width > 0 && size.height > 0) {
```

Comment:
You can probably do this because `getHeightWidth()` will return a zero size if the file is above the size threshold:

```suggestion
                        if (size.width > 0 && size.height > 0) {
                            this.hasRejectedTiff = true;
```